api/proto/config/devmodel.proto

```diff
@@ -47,19 +47,19 @@ message sWAdapterParams {
 
 // systemAdapters, are the higher l2 concept built on physicalIOs.
 // systemAdapters, gives all the required bits to turn the physical IOs
-// into useful IP endpoints
+// into useful IP endpoints.
 // These endpoints can be further used to connect to controller or
 // can be shared between workload/services running on the node.
 message SystemAdapter {
         // name of the adapter; hardware-specific e.g., eth0
         string name = 1;
 
         // deprecated; need level 2 spec. sWAdapterParams allocDetails = 20;
-
         // this is part of the freelink group
-        // deprecate: have a separate device policy object in the API
+        // DEPRECATED by PhyIoAdapter.usagePolicy.
         bool freeUplink = 2;
 
+        // uplink - DEPRECATED by PhysicalIO.Usage / PhysicalIO.UsagePolicy
         // this is part of the uplink group
         // deprecate: have a separate device policy object in the API
         bool uplink = 3;
```

Comment:
Ditto - does anything in EVE refer to this field? If not, how about commenting it out?